### PR TITLE
[#9100] Release V6.11.0

### LINKED ISSUE #9100: Release V6.11.0
Scheduled for Sunday, Sep 9, 2018, 5.00pm SGT.

### PR DESCRIPTION
**Pull request by xpdavid**
Fixes #9100

### CODE REVIEW

**Comment by damithc** [comment]
May want to hold this release back until #9109 is fixed

File: src/main/webapp/data/developers.json [hunk 1]
```diff
@@ -221,6 +221,9 @@
       "name": "Bhargav Thanki",
       "username": "bmthanki"
     },
+    {
+      "username": "bhumisaxena0223"
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: https://github.com/TEAMMATES/teammates/pull/9086#issuecomment-419650809

Comment (xpdavid) [reply src/main/webapp/data/developers.json]: Working on it 😄

File: src/main/webapp/data/developers.json [hunk 2]
```diff
@@ -1196,6 +1204,10 @@
       "name": "Shivansh Soni",
       "username": "shivanshsoni"
     },
+    {
+      "name": "Nicolás Marzano",
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: I think this should be much further up, considering it starts with N...

**Pull request merged**